### PR TITLE
Updated German translation

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: recovery/translation_de.ts [hunk 1]
```diff
@@ -11,42 +11,42 @@
...
         <location filename="backupthread.cpp" line="137"/>
         <source>%1: Updating partitions.json</source>
-        <translation type="unfinished"></translation>
+        <translation>partitions.json wird aktualisiert</translation>
     </message>
     <message>
         <location filename="backupthread.cpp" line="164"/>
         <source>%1: Writing image (%2)</source>
-        <translation type="unfinished"></translation>
+        <translation>Image wird geschrieben ($2)</translation>
```
Comment (procount) [inline recovery/translation_de.ts]: This one doesn't look quite right.
%1 is missing.
$2 needs changing to %2

**Comment by ghost** [comment]
I updated it.

**Comment by ghost** [comment]
I just added more translations.

**Comment by procount** [comment]
Thank you.

File: recovery/translation_de.ts [hunk 2]
```diff
@@ -11,42 +11,42 @@
...
         <location filename="backupthread.cpp" line="106"/>
         <source> :BTRFS file format</source>
-        <translation type="unfinished"></translation>
+        <translation> :BTRFS Dateiformat</translation>
     </message>
     <message>
         <location filename="backupthread.cpp" line="137"/>
         <source>%1: Updating partitions.json</source>
-        <translation type="unfinished"></translation>
+        <translation>partitions.json wird aktualisiert</translation>
```
Comment (lurch) [inline recovery/translation_de.ts]: Also missing `%1` ?

Comment (lurch) [reply recovery/translation_de.ts]: @procount Although I guess if you made it `%1: Updating %2` then you'd only need a single string-translation for both the `partitions.json` and `os.json` lines :slightly_smiling_face: (and the filenames themselves don't need translating anyway)

Comment (procount) [reply recovery/translation_de.ts]: That is very true. I'll try and remember to pick that up as I go through all the translations according to your previous comments on the use of .arg. :wink:

**Comment by RaspberryPiFan** [comment]
I  can't fix the missing %1 in my translation right now. I will fix it in 1 hour.

**Comment by RaspberryPiFan** [comment]
I  can't fix the missing %1 in my translation right now. I will fix it in 1 hour.

**Comment by RaspberryPiFan** [comment]
I  can't fix the missing %1 in my translation right now. I will fix it in 1 hour.

**Comment by RaspberryPiFan** [comment]
I  can't fix the missing %1 in my translation right now. I will fix it in 1 hour.

**Comment by procount** [comment]
Oops! I saw 4 identical comments, so I deleted 1 and they all disappeared 🤷‍♂️ 
Anyway, not to worry - no rush on the fix 😄 . Thanks for your help.

**Comment by procount** [comment]
Included in v3.2.1, but I think there maybe some more translations to do now, if you feel like it?

**Pull request closed without merging**